app/controllers/api/repositories_controller.rb

```diff
@@ -15,7 +15,12 @@ def projects
 
   def dependencies
     repo_json = RepositorySerializer.new(@repository).as_json
-    repo_json[:dependencies] = map_dependencies(@repository.repository_dependencies.includes(:project, :manifest) || [])
+    repository_dependencies = RepositoryDependency
+      .where(repository: @repository)
+      .where(manifest: @repository.manifests.order(created_at: :desc).limit(10))
```

Comment:
10 was chosen because it's used here:
https://github.com/librariesio/libraries.io/blob/7e4ee297bbbe929fcedea66c7ad16a727256e131/app/controllers/repositories_controller.rb#L40